<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>zamianajednostek</class>
 <widget class="QDialog" name="zamianajednostek">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Zamiana jednostek</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="3" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QPushButton" name="pushButton_wroc">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Wróć</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>mg/dl</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>g/dl</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="3" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Zamiana jednostek</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QPushButton" name="pushButton_wyzeruj">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Wyzeruj</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>mmol/l</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QPushButton" name="pushButton_oblicz">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Oblicz</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2" colspan="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="2" colspan="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="2" colspan="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLineEdit" name="lineEdit_1">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
